--- a/Game/MainClass/CloudberryKingdom.cs
+++ b/Game/MainClass/CloudberryKingdom.cs
@@ -28,7 +28,7 @@
         // Set this to true if you have all the resources for the full CK release.
         // Set this to false otherwise.
         // The game will crash if you attempt to load resources but do not have them.
-        public const bool LoadResources = true;
+        public const bool LoadResources = false;
 
         // Steam Integration
         // Set this to true to turn on Steam Integrtaion code
